composer.json

```diff
@@ -22,12 +22,15 @@
     ],
     "require": {
         "prewk/result": ">= 1.2.0",
-        "php": ">=7.0"
+        "php": ">=7.3.0"
     },
     "require-dev": {
-        "phpspec/phpspec": "^3.4.3",
-        "leanphp/phpspec-code-coverage": "^3.1.1",
+        "phpspec/phpspec": "^7.0.1",
+        "friends-of-phpspec/phpspec-code-coverage": "^6.0.0",
+        "phpunit/php-code-coverage": "^9.2.5",
+        "phpunit/php-file-iterator": "^3.0.5",
```

Comment:
I had to pin this sub-dependencies to a minimum version because bugs in them in lower versions would make the lowest builds fail.